src/Resources/views/Common/_addToWishlist.html.twig

```diff
@@ -0,0 +1,24 @@
+{% if injectWishlists()|length < 2 %}
```

Comment:
What does it do?

Reply:
If the user does not use the multiwishlist (didn't create new wishlsit before), then he adds "traditionally" the products to the wishlsit by click on btn 'add to wishlist'.

Reply:
So rename it accordingly. Method name suggests that it changes the state of application from template what should be avoided at all cost.